[IMP] inventory: thd batch updates (#2556)
odoo/documentation: content/applications/inventory_and_mrp/inventory/management/warehouses/warehouses_locations.rst

===============================
Manage Warehouses and Locations
===============================

Terminology
===========

Warehouse
---------

In Odoo, a **Warehouse** is the actual building/place in which your items are stocked. You can
setup multiple warehouses and create moves between warehouses.

Location
--------

> A **Location**, is a specific space within your warehouse. It can be considered as a sublocation of
> your warehouse, as a shelf, a floor, an aisle, etc. Therefore, a location is part of one warehouse
> only and it is not possible to link one location to multiple warehouses. You can configure as many
> locations as you need under one warehouse.

Reviewer: ```suggestion
A **Location** is a specific space within the warehouse. It can be a sublocation of the warehouse
(a shelf, a floor, an aisle, and so on). Therefore, a location is part of one warehouse only and it
is not possible to link one location to multiple warehouses. In Odoo, as many locations can be
configured as needed under one warehouse.
```

Left the bold formatting on `Location` instead of changing it to a `:guilabel:` because here, `Location` is being introduced as a term, not a GUI element that the reader must interact with.

> There are 3 types of locations:

Reviewer: ```suggestion
There are three types of locations:
```

> - The **Physical Locations** are internal locations that are part of the warehouses for which you
>   are the owner. They can be the loading and unloading area of your warehouse, a shelf or a
>   department, etc.

Reviewer: ```suggestion
- The **Physical Locations** are internal locations that are part of the warehouses that the company
  owns. They can be the loading and unloading areas of the warehouse, a shelf, a department, etc.
```

> - The **Partner Locations** are spaces within a customer and/or vendor's warehouse. They work the
>   same way as Physical Locations with the only difference being that you are not the owner of the
>   warehouse.

Reviewer: ```suggestion
- The **Partner Locations** are spaces within a customer and/or vendor's warehouse. They work the
  same way as physical locations, with the only difference being that they are not owned by the user's
  company.
```

> - The **Virtual Locations** are places that do not exist, but in which products can be placed when
>   they are not physically in an inventory yet (or anymore). They come in handy when you want to
>   place lost products out of your stock (in the **Inventory loss**), or when you want to take into
>   account products that are on their way to your warehouse (**Procurements**).

Reviewer: ```suggestion
- The **Virtual Locations** are places that do not exist, but in which products can be placed when
  they are not physically in an inventory yet (or anymore). They come in handy when recording lost
  products (**Inventory Loss**), or accounting for products that are on their way to the warehouse
  (**Procurements**).
```

> In Odoo, locations are structured hierarchically. You can structure your locations as a tree,
> dependent on a parent-child relationship. This gives you more detailed levels of analysis of your
> stock operations and the organization of your warehouses.

Reviewer: ```suggestion
In Odoo, locations are structured hierarchically. Locations can be structured as a tree, dependent
on a parent-child relationship. This gives more detailed levels of analysis of the stock operations
and the organization of the warehouses.
```

Configuration
=============

> To activate the locations, go to :menuselection:`Configuration --> Settings` and enable **Storage
> Locations**. Then, hit *Save*.

Reviewer: ```suggestion
To activate locations, go to :menuselection:`Configuration --> Settings` and enable
:guilabel:`Storage Locations`. Then, click :guilabel:`Save`.
```

.. image:: warehouses_locations/storage-location-warehouse-setting.png
   :align: center
   :alt: Enable the storage location feature in Odoo Inventory settings.

.. important::

>    If you want to manage several routes within your warehouses, then enable **Multi-Step Routes**
>    and check :doc:`../../routes/concepts/use-routes`.

Reviewer: ```suggestion
   To manage several routes within the warehouses, also enable :guilabel:`Multi-Step Routes` and check
   :doc:`../../routes/concepts/use-routes`.
```

> Creating a new warehouse
> ========================

Reviewer: ```suggestion
Create a new warehouse
======================
```

To create a warehouse, go to :menuselection:`Configuration --> Warehouse Management --> Warehouses`

> and click on **Create**.

Reviewer: ```suggestion
and click on :guilabel:`Create`.
```

> Fill out a **Warehouse Name** and a **Short Name**. The short name is five characters maximum.

Reviewer: ```suggestion
Then, fill out a :guilabel:`Warehouse Name` and a :guilabel:`Short Name`. The short name is five
characters maximum.
```

.. image:: warehouses_locations/create-new-warehouse.png
   :align: center
   :alt: Short name field of a warehouse on Odoo Inventory.

.. important::

>    The **Short Name** appears on your transfer orders and other warehouse documents. We recommend
>    using an understandable one (e.g.: WH/[first letters of location]).

Reviewer: ```suggestion
   The :guilabel:`Short Name` appears on transfer orders and other warehouse documents. Odoo recommends
   using an understandable one like "WH/[first letters of location]".
```

> Now, go back to your dashboard. There, new operations related to your warehouse have been
> automatically generated.

Reviewer: ```suggestion
Now, go back to the :guilabel:`Inventory` dashboard. There, new operations related to the newly
created warehouse have been automatically generated.
```

.. image:: warehouses_locations/new-transfer-types.png
   :align: center
   :alt: Inventory app dashboard displaying new transfer types for the recently created warehouse.

.. note::

>    Adding a second warehouse will automatically activate the **Locations** setting.

Reviewer: ```suggestion
   Adding a second warehouse will automatically activate the :guilabel:`Locations` setting.
```

> Creating a new location
> =======================

Reviewer: ```suggestion
Create a new location
=====================
```

To create a location, go to :menuselection:`Configuration --> Warehouse Management --> Locations`

> and click on **Create**.

Reviewer: ```suggestion
and click on :guilabel:`Create`.
```

> Fill out a **Location Name** and a **Parent Location** and hit *Save*.

Reviewer: ```suggestion
Then, fill out a :guilabel:`Location Name` and a :guilabel:`Parent Location` and click
:guilabel:`Save`.
```
